backend/tests/Notifo.Domain.Tests/Integrations/Discord/DiscordTests.cs

```diff
@@ -60,14 +80,15 @@ public async Task Should_send_full_message()
     [Fact]
     public async Task Should_fail_on_user()
     {
-        // Random 18-digit number
-        var random = new Random();
-        string userId = string.Join(string.Empty, Enumerable.Range(0, 18).Select(number => random.Next(0, 9)));
+        var invalidUser = Guid.NewGuid().ToString();
```

Comment:
Hey Sebastian,
I think that you can't use the Guid here since Discord user id is all digits (without any letters) - that's why I didn't use GUID but instead did that 18 digit number generation